src/nomics_tests.py

```diff
@@ -33,7 +34,22 @@ def test_candles():
         return True
 
 
+def test_pairs():
+    for chain_id in nomics["chain"].keys():
+        for exchange in nomics["chain"][chain_id]["exchanges"]:
+            for market in nomics["chain"][chain_id]["exchanges"][exchange]["markets"].keys():
+                hash = nomics["chain"][chain_id]["exchanges"][exchange]["markets"][market]
+                print(chain_id, exchange, market, hash)
+    start = "2022-01-18"
+    end = "2022-01-19"
+    query = get_candles(exchange, hash, start, end)
```

Comment:
what does this do? `query` will always only get called with a single `exchange` and `hash` here, right? whatever happens to be the last value of `exchange` and `hash` when executing the above loops. is that the intent?

Reply:
Ahh, i didn't want to spend too many queries doing it for all of them, so really it's just dumping out the dictionary and checking the last one didn't give an error.